01week/rockPaperScissors.js

```diff
@@ -9,9 +9,37 @@ const rl = readline.createInterface({
 
 
 function rockPaperScissors(hand1, hand2) {
+  hand1 = hand1.toLowerCase().trim();
+  hand2 = hand2.toLowerCase().trim();
 
-  // Write code here
-
+  
+    //If User1 input is 'rock' and User2 input is 'scissor', User1 wins. 
+    if (hand1 === "rock" && hand2=== "scissors")
+    return "Hand one wins!"
+    //If User1 input is 'rock' and User2 input is 'paper', User2 wins.
+    else if (hand1 === "rock" && hand2=== "paper")
+    return "Hand two wins!"
+    //If User1 input is 'rock' and User2 input is 'rock', it's a tie.
+    else if (hand1 === "rock" && hand2=== "rock")
+    return "It's a tie!"
+    //If User1 input is 'paper' and User2 input is 'rock', User1 wins.
+    else if (hand1 === "paper" && hand2=== "rock")
+    return "Hand one wins!"
+    //If User1 input is 'paper' and User2 input is 'scissors', User2 wins.
+    else if (hand1 === "paper" && hand2=== "scissors")
+    return "Hand two wins!"
+    //If User1 input is 'paper' and User2 input is 'paper', it's a tie.
+    else if (hand1 === "paper" && hand2=== "paper")
+    return "It's a tie!"
+    //If User1 input is 'scissors' and User2 input is 'paper', User1 wins.
+    else if (hand1 === "scissors" && hand2=== "paper")
+    return "Hand one wins!"
+    //If User1 input is 'scissors' and User2 input is 'rock', User2 wins.
+    else if (hand1 === "scissors" && hand2=== "rock")
+    return "Hand two wins!"
+    //If User1 input is 'scissors' and User2 input is 'scissors', it's a tie
+    else if (hand1 === "scissors" && hand2=== "scissors")
+    return "It's a tie!"
 }
```

Comment:
How might you write one else if for all ties? Again, thank you for all the comments, super readable.

Reply:
Thank you for your help Katie! Would I write it like this?  
function rockPaperScissors(hand1, hand2) {
else  if(hand1 === hand2){
return 'That is a tie!';